client/src/pages/MoviesList/styled.js

```diff
@@ -55,3 +55,64 @@ export const SuccessBanner = styled.img.attrs({
 })`
   position: absolute;
 `
+export const TurnsContainer = styled.div`
```

Comment:
it's ok to use divs, but whenever possible, you should try to use the html element that most closely matches what you're building. The turns table is technically a table, so it's technically more semantically correct to use `<table>` elements.

This is important for accessibility, which we don't really care bout right now, but good to know in interviews or if you're working for a company that cares a lot about accessibility.